spec/takeaway_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'takeaway'
+
+describe Takeaway do 
+
+subject(:takeaway) {described_class.new}
+let(:dish1) {double(:dish1)} 
+let(:dish2) {double(:dish2)} 
+
+
+  describe "#list_menu" do 
+ 
+    it 'displays a list of dishes' do 
+    expect(takeaway.list_menu).to include ("Roast Pork Belly Hirata Buns")
+    end
+
+  end
+
+  describe "#select_item" do 
+
+    it { should respond_to(:select_item).with(2).arguments }
+
+    it 'allows you to select a dish from the menu' do 
+    takeaway.select_item(dish1)
+    expect(takeaway.order).to include (dish1)
+    end
+
+    it 'allows you to select multple items from the menu' do
+    takeaway.select_item(dish1)
+    takeaway.select_item(dish2, 2)
+    takeaway.select_item(dish1)
+    expect(takeaway.order).to eq [dish1, dish2, dish2, dish1]
+    end
+  
+    it 'confirms the items you have selected' do 
+    expect(takeaway.select_item(dish1,2)).to eq "you added 2 #{dish1} to your basket"
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)